frontend/packages/dev-console/src/components/add/AddPageLayout.tsx

```diff
@@ -66,7 +65,6 @@ const AddPageLayout: React.FC<AddPageLayoutProps> = ({ title, hintBlock: additio
             <div className="odc-add-page-layout__resource-quota-message-block">
               <ResourceQuotaAlert namespace={activeNamespace} />
             </div>
-            <RestoreGettingStartedButton userSettingsKey={GETTING_STARTED_USER_SETTINGS_KEY} />
```

Comment:
Can you please remove the complete component since it's not used anymore?

Reply:
`RestoreGettingStartedButton ` is used in kubevirt plugin code. We can keep this until kubevirt is moved completely to dynamic plugin.